### DISCUSSION TITLE
User friendly settings index?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#517] [Q&A] [answered]
Cruiser has many settings and functions - This is great and thanks for that. However sometimes I'm really overwhelmed and especially having issues to find the right setting (at the right place), I'm looking for in the right (sub) menu. And I new I had seen it once.

Example: 
I know in advance I had seen the setting before but where was it: _"turning speed cam on/off"_ It really took me a while to explore it in the right sub menu again.

Solution:
Having a (scrollable) menu page which has "catchwords" like "speed cam" or "3D buldings" etc. where the menu either display where to find it like:

**Speed Cam**
 _settings/navigation/nearby POIs..._

or even better, a link which jumps directly to the relevant submenu.

**Comment by 0709wiwiwi** [comment, marked as answer]
Go to.
Hamburger menu | Settings | Tap search Icon | Type a keyword like =  Speed ca....

This very handy search tool could be more user friendly be presented
- This by a **Question Mark Icon** at the bottom of the diverse menus

**Reply by DIrgi** [reply]
Cool, thx for the tip - sometimes the easy things are just around the corner

**Comment by devemux86** [comment]
Alternatively, the search of the settings could be as a separate large button at the top.

Cruiser supports both implementations: search in top bar or search via extra button
And they work the same, click on the results and the relevant settings screen opens.

<img src="https://github.com/devemux86/cruiser/assets/3484020/1e8757f3-17e5-4801-8283-0e8005ba2776" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/7993b48d-1066-4433-aa27-ae9dda3b9144" width="50%"/>

**Reply by devemux86** [reply]
Although it currently has the same icon / title as the normal search settings found below.

**Reply by bios4** [reply]
...which COULD lead to confusion!?

**Reply by devemux86** [reply]
If we want to use the separate search button, then there will be changes to the icon / title on one of the two items.

**Reply by DIrgi** [reply]
I think the(my) issue was to simply do not see the search function to solve my request. Usually on reason is that quite often search functions are lousy and don’t really help you and second I didn’t really connect the search function with finding menue settings.
Maybe this happens to others as well!?

A good support can be maybe, a renaming of the „just“ „search“ into:“Search for settings and others(or anything)“

**Reply by devemux86** [reply]
Apps that have search in their settings usually have it on the top bar so it doesn't take up space in the settings area.
(like now in Cruiser)
